devsetup/tripleo/tripleo_install.sh

```diff
@@ -157,10 +157,18 @@ fi
 # defaults for non-ceph case
 CEPH_OVERCLOUD_ARGS=""
 ROLES_FILE="/home/zuul/overcloud_roles.yaml"
+if [ "$EDPM_COMPUTE_CEPH_ENABLED" = "true" ] ; then
+    ROLES_FILE="/home/zuul/roles.yaml"
+fi
+if [ "$TRIPLEO_NETWORKING" != "true" ] ; then
+    # disable external gateway for controller nodes
+    sed -i "s/default_route_networks: \['External'\]/default_route_networks: \['ControlPlane'\]/" $ROLES_FILE
+    sed -i "/External:/d" $ROLES_FILE
+    sed -i "/subnet: external_subnet/d" $ROLES_FILE
+fi
```

Comment:
please lets ensure we have green run for the ceph and non ceph multinode jobs with this.

WRT the commit message .... the external network *is* now in the roles file used by the mutlinodes https://github.com/marios/install_yamls/blob/b73958657270d3ab2be53f2c0a23b889326bdb35/devsetup/tripleo/overcloud_roles.yaml#L20

Reply:
(I checked there https://review.rdoproject.org/r/c/testproject/+/53212/21#message-5344b7631720caf2d94efe0ac60119e3114ec8e2)

Reply:
The external network is being removed with the deploy script, so it is not really used in CI.
This patch makes it configurable, as local deploys gonna need it. I've also analyzed already the multinode testproject, please see the outcomes - tripleo network config ping tests worked OK. I can recheck it though, to get it till completition, if we want to be 100% on the safe side

Reply:
rechecked https://review.rdoproject.org/r/c/testproject/+/54199

Reply:
hm , tripleo deployment fails "TASK | Check password of Keystone user" with this change

Reply:
perhaps we need this https://review.rdoproject.org/r/c/rdo-jobs/+/55215